Ziti.NET.Standard/src/OpenZiti/API.cs

```diff
@@ -239,4 +239,13 @@ public struct posture_query {
     public struct ziti_process {
         public string path;
     }
+
+    public struct ziti_identity

```

Comment:
probably can remove this? i don't think it's necessary?

Reply:
I am using this struct to fetch the name of the identity inside the event_cb function 
ziti_identity zid = Marshal.PtrToStructure<ziti_identity>(idPtr)